Thomah/boothby
# Add default user & add users table

## Status
**READY

## Migrations
NO

## Description
2 main tasks in this PR : 
- When starting the server, we check if there are existing users in DB, if not, we create a default admin user, which can be configured in the set-env.sh file.
- When logged, we can access the users menu : On this one, there is a menu which lists all users, with buttons enabling the removal of those users. (and there is a still the add user functionnality)
PS : We CAN'T delete a user if he is the last one in the DB. There must always be one user.

![image](https://user-images.githubusercontent.com/8972383/54134955-2ceb4c80-4419-11e9-9694-65bd8b96cb87.png)

## Related issues
- #28 
- #36 
- #37 

## Steps to Test or Reproduce
Just start the server, and go to the users menu. You could also delete all users in your db, to see that the the default user is well created. 

```sh
git checkout add_default_user
```
